docs/extend/extend-apiml/api-mediation-infinispan.md

```diff
@@ -16,38 +18,44 @@ For more information on Infinispan, see the [official Infinispan documentation](
+you will have to specify all the cluster nodes (members). Each Infinispan node is bound to specific Caching Service instance and runs on a different port and host, which can be configured. See the [infinispan configuration](#infinispan-configuration) to know how to configure multiple Infinispan nodes.
 
 For more information on Infinispan replication and how to configure a replica instance, see the [official Infinispan Cross-site Replication documentation](https://infinispan.org/docs/stable/titles/xsite/xsite.html).
 
 ## Infinispan configuration
 
-Configure Infinispan as a storage solution through the Caching service by setting the following environment variables. Environment variables can be set by adding them to `instance.env`.
+Configure Infinispan as a storage solution through the Caching service by setting the following configuration parameters in the `zowe.yaml`.
  
-* **`CACHING_STORAGE_INFINISPAN_INITIALHOSTS`**
+* **`zowe.components.caching-service.storage.infinispan.initialHosts`**
 
   This property specifies the list of cluster nodes (members). In case of multiple instances, the value for each Caching Service instance can be 
-  either a list of all the members, separated by a comma, or just the replica. The format is `${JGROUPS_BIND_ADDRESS}[${JGROUPS_BIND_PORT}]`.
+  either a list of all the members, separated by a comma, or just the replica. The format is `${haInstance.hostname}[${zowe.components.caching-service.storage.infinispan.jgroups.port}]`.
 
-  **Example:**
-  `CACHING_STORAGE_INFINISPAN_INITIALHOST=caching-service[7600]`
 
-
-* **`CACHING_STORAGE_INFINISPAN_PERSISTENCE_DATALOCATION`**
+* **`zowe.components.caching-service.storage.infinispan.persistence.dataLocation`**
 
   The path where the Soft-Index store will keep its data files for the Infinispan Soft-Index Cache Store. 
   The default value is `data`. If you run the Caching Service in HA and the instances use the same filesystem,
-  you have to specify a different value of the `CACHING_STORAGE_INFINISPAN_PERSISTENCE_DATALOCATION` property for each
+  you have to specify a different value of the `zowe.components.caching-service.storage.infinispan.persistence.dataLocation` property for each
   instance. For more information, see [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
 
 
-* **`JGROUPS_BIND_PORT`**
-
-  The Jgroups port used by Infinispan.
-
+* **`zowe.components.caching-service.storage.infinispan.jgroups.port`**
 
-* **`JGROUPS_BIND_ADDRESS`**
+  The port number used by Infinispan to synchronise data among cahing-service instances.
 
-  The Jgroups address used by Infinispan.
 
-  
+  **Example of Caching service configuration using Infinispan:**
+    ```
```

Comment:
```suggestion
    ```yaml
```

Reply:
this would produce ```yaml in the docs website, is that correct?

Reply:
It's because of identation, leave one empty line before and align the ``` to the line above (at `**`)

Reply:
thanks